src/api-transition-helpers.coffee

```diff
@@ -0,0 +1,17 @@
+LineNumberGutterName = 'line-numbers'
+
+# Given decorationParams/Properties from the 1.0 API, applies changes to enable
+# the 2.0 API under the covers. In 2.0, the `trueType` property will become `type`.
+translateDecorationParamsOldToNew = (decorationParams) ->
```

Comment:
> On second thought, the reason I wanted to do the isTrueType is to avoid having places in the Atom Core code where checking for type='line-gutter' will work. Everything should instead conform to the new API, which requires type='gutter' and the name.

The extra complexity in terms of stored state is setting off alarm bells for me. Perhaps we could translate `::getDecorations` params as well? In that case a class method on decoration could make it available outside so we don't need an extra helper to be extracted.